client/src/main/java/com/defold/extender/client/ExtenderClient.java

```diff
@@ -141,6 +155,89 @@ private static Set<String> getCachedFiles(String json) throws ExtenderClientExce
         return cachedFiles;
     }
 
+
+    private void build_async(String platform, String sdkVersion, MultipartEntity entity, File destination, File log) throws ExtenderClientException {
+        try {
+            String url = String.format("%s/build_async/%s/%s", extenderBaseUrl, platform, sdkVersion);
+            HttpPost request = new HttpPost(url);
+            request.setEntity(entity);
+
+            print("Sending async build request to %s", url);
+
+            addAuthorizationHeader(request);
+
+            AbstractHttpClient client = new DefaultHttpClient();
+            client.setCookieStore(httpCookies);
+            HttpResponse response = client.execute(request);
+
+            final int statusCode = response.getStatusLine().getStatusCode();
+            if (statusCode == HttpStatus.SC_OK) {
+                String jobId = EntityUtils.toString(response.getEntity());
+                print("Async build request was accepted as job %s", jobId);
+                long currentTime = System.currentTimeMillis();
+                Integer jobStatus = 0;
+                Thread.sleep(buildSleepTimeout);
+                while (System.currentTimeMillis() - currentTime < buildResultWaitTimeout) {
+                    print("Checking job status for job %s", jobId);
```

Comment:
I consider this very verbose information.
I don't want us end up in a situation where we cannot see our warnings and errors due to the amount of other info.

Reply:
Removed